arenadata/readme.md

```diff
@@ -32,6 +32,27 @@ We need to execute [../concourse/scripts/ic_gpdb.bash](../concourse/scripts/ic_g
 
 To use gdb inside the container, add the `--privileged` flag to the run command.
```

Comment:
This line and two above are for all regression tests. Move it under new regression tests.

Reply:
Done